src/Spice86.Core/Emulator/InterruptHandlers/Dos/DosInt21Handler.cs

```diff
@@ -108,6 +108,15 @@ public class DosInt21Handler : InterruptHandler {
         AddAction(0x4F, () => FindNextMatchingFile(true));
         AddAction(0x51, GetPspAddress);
         AddAction(0x62, GetPspAddress);
+        AddAction(0x63, GetLeadByteTable);
+    }
+
+    private void GetLeadByteTable() {
```

Comment:
What is this table ? What is its purpose ? Is it entirely implemented or os this a stub ? Why do we return 0 ?

Can all of this be documented in XML summary for this method ?

Also this method should be public, for overrides.

Reply:
Looks like this http://www.ctyme.com/intr/rb-3143.htm
More details here: http://www.ctyme.com/intr/rb-3142.htm#Table1746

Not sure why ES:BX is set while spec says DS:SI

Reply:
I added docs and implemented a dummy 0 return.

We'll pick this up further when we start supporting asian character sets in DOS